usr/share/rear/skel/default/etc/scripts/run-sshd

```diff
@@ -1,16 +1,27 @@
 #!/bin/bash
-# check inittab for "ssh:23:respawn:/bin/sshd -D"
+# Check /etc/inittab for "ssh:23:respawn:..." which means sshd should be started
+# (that entry is written by rescue/default/500_ssh.sh if sshd should be started):
 if grep -q '^ssh:' /etc/inittab ; then
```

Comment:
A bit strange to have a systemd-based initialization depend on init-based stuff, but also reflected in `usr/share/rear/skel/default/usr/lib/systemd/system/sshd.service`. Maybe something for a future cleanup.

Reply:
There is so much to clean up in ReaR, even crucial things, e.g.
https://github.com/rear/rear/pull/1603#issuecomment-347860869